Multisig/UI/Safe Management/Add Safe/EnterSafeNameViewController.swift

```diff
@@ -48,14 +48,14 @@ class EnterSafeNameViewController: UIViewController {
     @objc private func didTapNextButton() {
         guard let name = name, let address = address else { return }
         Safe.create(address: address.checksummed, name: name)
-        if let _ = App.shared.settings.signingKeyAddress {
-            completion()
-        } else if !AppSettings.hasShownImportKeyOnboarding {
+        if !AppSettings.hasShownImportKeyOnboarding && App.shared.settings.signingKeyAddress == nil {
             let vc = SafeLoadedViewController()
             vc.completion = completion
             vc.hidesBottomBarWhenPushed = true
             show(vc, sender: self)
             AppSettings.hasShownImportKeyOnboarding = true
+        } else {
```

Comment:
@sche this will solve the bug #894 also, please just close the issue once you merge it